src/components/Navbar/Rightbar.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Menu,Link } from "antd";
+
+const Rightbar = ({ mode }) => {
+    const items =[
+        {
+            label: (
+                <Link to = {'/'}></Link>
```

Comment:
The items array is the same as in the `Leftbar` , can you extract it so you will use it once ?